Tests/PINRemoteImage-tvOSTests-Bridging-Header.h

```diff
@@ -0,0 +1,4 @@
+//
+//  Use this file to import your target's public headers that you would like to expose to Swift.
```

Comment:
Is this file necessary?

Reply:
It was auto created by Xcode when I added the Swift test, should I try to remove it?

Reply:
I guess if it still is working without it 😅

Reply:
It turns out it is needed for the tests to run on tvOS target